modules/imapauth/lib/Auth/Source/MyAuth.php

```diff
@@ -0,0 +1,173 @@
+<?php
+class sspmod_imapauth_Auth_Source_MyAuth extends sspmod_core_Auth_UserPassBase {
```

Comment:
I've started to work on new module called unixauth which implements a authentication with a few protocols (at the beginning SSH and IMAP) and tried to use "a new namespace schema" but got an error:

<?php
namespace SimpleSAML\Module\unixauth\Auth\Source;
class UNIX extends sspmod_core_Auth_UserPassBase {

error:
PHP Fatal error:  Class 'SimpleSAML\\Module\\unixauth\\Auth\\Source\\sspmod_core_Auth_UserPassBase' not found in /usr/share/simplesamlphp/modules/unixauth/lib/Auth/Source/MyAuth.php on line 7, referer: https://192.168.110.3/simplesamlphp/module.php/core/authenticate.php

I don't quite understand what's wrong